Student-Class/src/student/pkgclass/StudentClass.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package student.pkgclass;
+
+/**
+ *
+ * @author Ziying Guo
+ */
+public class StudentClass {

```

Comment:
This isn't a student class. This is an application that **uses** the `Student` class. You can name this class something like `StudentApplication`. This class is just meant to have a `main` method.